packages/backend/src/assets/ai.json

```diff
@@ -83,7 +83,8 @@
       "hw": "CPU",
       "registry": "Hugging Face",
       "license": "Apache-2.0",
-      "url": "https://huggingface.co/TheBloke/Mistral-7B-Instruct-v0.1-GGUF/resolve/main/mistral-7b-instruct-v0.1.Q4_K_M.gguf"
+      "url": "https://huggingface.co/TheBloke/Mistral-7B-Instruct-v0.1-GGUF/resolve/main/mistral-7b-instruct-v0.1.Q4_K_M.gguf",
+      "memory": 4370464768
```

Comment:
4,07 * 1024 * 1024 * 1024 = 4 370 129 224 after rounding on my side

Reply:
oh right, I've first rounded at the MiB, then converted to bytes. That should be ok now. Thanks